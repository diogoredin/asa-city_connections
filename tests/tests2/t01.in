10
10
3 3
7 10
2 3
4 8
8 8
6 10
1 2
9 4
5 10
10 3
10
3 5 7
1 9 7
7 2 1
3 1 6
7 5 9
6 9 3
2 9 10
2 10 7
4 6 10
4 8 7
