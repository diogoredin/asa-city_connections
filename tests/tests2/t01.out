41
6 4
